test/functional/api/v1/owners_controller_test.rb

```diff
@@ -275,10 +275,10 @@ def self.should_respond_to(format)
           end
 
           should "include mfa warning in response" do
-            expected_response = "#{@second_user.handle} was added as an unconfirmed owner. Ownership access will be enabled after the user clicks on the confirmation mail sent to their email.\n\n[WARNING] For protection of your account and gems, we encourage you to set up multifactor authentication " \
-            "at https://rubygems.org/multifactor_auth/new. Your account will be required to have MFA enabled in the future."
+            mfa_recommended_warning = "\n\n[WARNING] For protection of your account and gems, we encourage you to set up multifactor authentication " \
+                                "at https://rubygems.org/multifactor_auth/new. Your account will be required to have MFA enabled in the future."
```

Comment:
I don’t understand what’s happening with the string literal indentation here & elsewhere. It doesn’t line up with anything; is it meant to?

Reply:
With all the indentation changes, it was a result of rubocop autocorrect. It looks like this one slipped by...